content/app/guides/multi-app-solution/_index.en.md

```diff
@@ -0,0 +1,64 @@
+---
+title: General approach for making a multi-app solution in Altinn
+linktitle: Multi-app solution
+description: Considerations and explanations of how to go about when creating a multi-app solution
+weight: 250
+aliases:
+
+- /app/multi-app-solution/
+
+---
+
+Before continue reading these guidelines, please consider if
+a multi-app solution is what you need to fulfill the purpose
+of your form(s).
+
+## What is a multi-app solution?
+
+A multi-app solution is a solution consisting of two or more
+cooperating apps, where typically (as per now) the "
+original" application(s) will trigger the creation of a new
```

Comment:
the alternate wordings make the text harder to read. Focus on a basic case and assume that the reader understands the concept well enough to extend it to multiple cooperating apps 
```suggestion
original" application will trigger the creation of a new
```